### PR TITLE
Fixes #22099 - reuse core exporting

### PR DESCRIPTION
**Pull request by ares**
Introduced in Foreman 1.12, see http://projects.theforeman.org/issues/14509 for more details

### CODE REVIEW

File: app/models/job_template.rb [hunk 1]
```diff
@@ -1,10 +1,10 @@
 class JobTemplate < ::Template
-  include ForemanRemoteExecution::Exportable
+  include ::Exportable
 
   class NonUniqueInputsError < Foreman::Exception
   end
 
-  attr_exportable :name, :job_category, :description_format, :snippet, :template_inputs,
+  attr_exportable :job_category, :description_format, :template_inputs,
```
Comment (ares) [inline app/models/job_template.rb]: note for reviewer: these are now inherited from Template along with `model`

**Comment by adamruzicka** [comment]
Shouldn't we keep the old tests just to be sure it behaves the same way as before?

**Comment by ares** [comment]
I think that would be a duplication of tests from core.

**Comment by adamruzicka** [comment]
@ares could you please rebase this?

**Comment by ares** [comment]
@adamruzicka sure, rebased

**Comment by ares** [comment]
rebased to avoid test failures (only postgres on 2.4, probably was not related)

**Comment by ares** [comment]
@adamruzicka this got stuck somewhere, mind to take another look?

**Comment by adamruzicka** [comment]
Sorry for the delay. I looked at it again now and it works like a charm, merged. Thanks @ares !

**Pull request merged**